config.go

```diff
@@ -180,6 +180,10 @@ func (m *Defender) UnmarshalJSON(b []byte) error {
 			URL: m.URL,
 		}
 	case "tarpit":
+		// check if tarpit config is empty
+		if rawConfig.TarpitConfig.Content == (tarpit.Content{}) {
```

Comment:
I intentionally made content optional so that the timeout reader would just be used to stall requests

Reply:
Ah.... interesting okay.